addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/XAspNetVersionScanRule.java

```diff
@@ -57,25 +57,18 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
             List<String> found = msg.getResponseHeader().getHeaderValues(header);
 
             if (!found.isEmpty()) {
-                this.raiseAlert(msg, id, found.get(0));
+                createAlert(
+                        Alert.CONFIDENCE_HIGH,
+                        Constant.messages.getString(MESSAGE_PREFIX + "refs"),
+                        found.get(0),
+                        Constant.messages.getString(MESSAGE_PREFIX + "soln"),
+                        Constant.messages.getString(MESSAGE_PREFIX + "desc")
+                )
+                .raise();
             }
         }
     }
 
-    private void raiseAlert(HttpMessage msg, int id, String evidence) {
```

Comment:
Do not move the method, also, static values (like description) should be the same for example alerts.